{{cookiecutter.project_name}}/notebooks/TriggerModelDeploy.py

```diff
@@ -65,3 +73,14 @@
     f"Successfully triggered model CD deployment workflow for {model_uri}. See your CD provider to check the "
     f"status of model deployment"
 )
+
+{% elif cookiecutter.cicd_platform == "azureDevOpsServices" -%}
+# COMMAND ----------
+from deploy import deploy
+deploy(model_uri, env)
```

Comment:
```suggestion
# TODO: Add any additional pre-deployment checks here to ensure model quality before calling
# `deploy` to push the model to the desired environment.
deploy(model_uri, env)
```

Maybe we can add a user-facing TODO here to make it clear where to add pre-deploy checks